### DISCUSSION TITLE
Backward compatibility with Discord (API)

### DISCUSSION THREAD

**Discussion by NEsanya** [stoatchat/discussions#253] [Feature Suggestions]
Hi there!

Lately I've been thinking about discord backward compatibility a lot. How much is it possible? Write all your ideas here :)

**Comment by Zomatree** [comment]
This wouldn't be possible as there is a lot of stuff that is very different making compatibility impossible.

**Reply by insertish** [reply]
To add on this:
At best there could be a separate compatibility layer which translates the API calls, but obviously not the entire Discord API or all of Revolt's features would be available.

**Reply by erkinalp** [reply]
@insertish Rather that all the features would translate but with different rules of operation. For example, Discord's explicit allow wins vs. Revolt's explicit deny wins permission system in groups.
